.gitignore

```diff
@@ -127,3 +127,5 @@ npm-debug.log
 nbproject
 .idea
 .node_history
+tests/e2e/screenshots
+tests/e2e/videos
```

Comment:
This prevents cypress from cluttering up git history with additional files for videos and screenshots